src/Symfony/Component/Validator/Mapping/ClassMetadata.php

```diff
@@ -262,6 +279,10 @@ public function addConstraint(Constraint $constraint)
      * @param Constraint $constraint The constraint
      *
      * @return ClassMetadata This object
+     *
+     * @throws GroupDefinitionException When the "class name" group has
```

Comment:
"default group" is the name used in the comments to refer to what you call "class name group". Also, `getDefaultGroup()` returns that group name. In the end, "Default" is just an alias of the default group (i.e. the "class name group").